modules/admin-ui/src/main/webapp/scripts/shared/partials/modals/event-details.html

```diff
@@ -1483,31 +1486,46 @@ <h3>{{ item.name }}</h3>
             <header translate="EVENTS.EVENTS.DETAILS.WORKFLOW_DETAILS.TITLE"><!-- Workflow Details --></header>
             <div class="obj-container">
               <table class="main-tbl">
-                <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.ID"><!-- ID --></td>
-                  <td>{{ subNavData.id }}</td>
-                </tr>
                 <tr>
                   <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.TITLE"><!-- Title --></td>
                   <td>{{ subNavData.title }}</td>
                 </tr>
+                <tr ng-if="subNavData.description">
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.DESCRIPTION"><!-- Description --></td>
+                  <td>{{ subNavData.description }}</td>
+                </tr>
                 <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.TYPE"><!-- Type --></td>
-                  <td>{{ subNavData.workflow }}</td>
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED_BY"><!-- Submitted by --></td>
+                  <td>
+                    {{ subNavData.creator.name }}
+                    <span ng-if="subNavData.creator.email">{{'<' + subNavData.creator.email + '>' }}</span>
+                  </td>
                 </tr>
                 <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STATE"><!-- State --></td>
-                  <td>{{ subNavData.state | translate }}</td>
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.SUBMITTED_AT"><!-- Submitted at --></td>
+                  <td>{{ subNavData.submittedAt | localizeDate : 'dateTime' : 'medium' }}</td>
                 </tr>
                 <tr>
-                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STARTED"><!-- Started --></td>
-                  <td>{{ subNavData.start | localizeDate : 'dateTime' : 'medium' }}</td>
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.STATUS"><!-- Status --></td>
+                  <td>{{ subNavData.status | translate }}</td>
+                </tr>
+                <tr ng-if="subNavData.status !== 'EVENTS.EVENTS.DETAILS.WORKFLOWS.OPERATION_STATUS.RUNNING'">
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.DURATION"><!-- Duration --></td>
+                  <td>{{ subNavData.executionTime | humanDuration }}</td>
+                </tr>
+                <tr with-role="ROLE_ADMIN">
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.ID"><!-- WIID --></td>
+                  <td>{{ subNavData.wiid }}</td>
+                </tr>
+                <tr with-role="ROLE_ADMIN">
+                  <td translate="EVENTS.EVENTS.DETAILS.WORKFLOWS.WDID"><!-- WDID --></td>
+                  <td>{{ subNavData.wdid }}</td>
                 </tr>
               </table>
             </div>
           </div>
 
-          <div class="obj tbl-details">
+          <div class="obj tbl-details" with-role="ROLE_ADMIN">
```

Comment:
Again, why only ROLE_ADMIN (not even an org admin)?

Reply:
Same as above: Only ROLE_ADMIN has access to the respective REST APIs (you could configure this differently, but seems common).

Reply:
I could imagine that e.g. an org admin would want to check which workflow was used for something and this is the only thing which identified the workflow definition. and more than ROLE_ADMIN has access to some API endpoints.

I agree that this is most useful for administrators, but that may also be org admins or again users reporting problems with a specific workflow.

Reply:
Again: Users - at least in our case - report problems on event level. They had the workflow instance ID for years now but there was no single case any kind of user would ever have contacted us referring to a workflow instance ID. I would be surprised if other Adopters would handle this different.

Reply:
@staubesv I could be reading this discussion incorrectly, but we run several types of workflows on events and the users/responsible non-admin personel definately want to know which workflows were run on an events, and which were not. They know to look at the details of the workflow so they can report on which operation caused an error. We do a hold-for-error-recovery on workflow errors, so it's important to get as much details from the users on the cause of the error as possible.

Reply:
@karendolan Unprivileged users can still see which workflows ran/run on an event as the workflow title is displayed. The users also still can look at the individual operations although I doubt that they will understand the error messages.
So they still can report which operation has failed - although this does not really provide more information as the event ID would have as it is easy to lookup the failed operation in the Admin UI anyway...

What unprivileged users (including tenant administrators) cannot do with this patch: They don't see the workflow instance ID, the workflow definition ID and the workflow configuration.

The motivation to hide this information are that...

- ... the workflow instance ID only makes sense if you - for whatever reason - would want to call REST endpoints for a given workflow instance ID
- ... the workflow definition ID and workflow configuration require access to / understand of the workflow definitions (etc/workflows/*.xml). Without that, you present information to users which doesn't help them much

Generally, Opencast tends to be very technical which to me seems the main reason  that people are afraid of adopting Opencast. This could be improved by not displaying technical details that require quite some Opencast knowledge to just every user just everywhere. That is the main motivation to hide this fields from non-admin users.

Reply:
> Unprivileged users can still see which workflows ran/run on an event as the workflow title is displayed.
No, they can't. They can only tell that *a* workflow ran on the event. They cannot necessarily say which one because that's identified by the workflow instance id only and they cannot necessarily tell which type of workflow was used because that's only identified by the workflow definition id. These are identifiers after all. Descriptions or titles can help but they are not unique.

I agree that all of this workflow stuff is technical but remember that all of this is accessible only for users which you explicitly provided with access to this technical information. If you are looking at users which only report that something went wrong on an event level, there is no reason to provide that priviledge in the first place, is there?